scripts/rabbitmq-service.bat

```diff
@@ -171,8 +178,27 @@ if exist "!RABBITMQ_CONFIG_FILE!.config" (
                                 -conf_advanced "!RABBITMQ_ADVANCED_CONFIG_FILE!" ^
                                 -config "!RABBITMQ_ADVANCED_CONFIG_FILE!"
     )
+) else (
+    if exist "!RABBITMQ_CONFIG_FILE!.config" (
+        set RABBITMQ_CONFIG_ARG=-config "!RABBITMQ_CONFIG_FILE!"
+    ) else if exist "!RABBITMQ_CONFIG_FILE!.config" (

```

Comment:
this `else if`  should be `if exist "!RABBITMQ_CONFIG_FILE!.conf"` as [`rabbitmq-server.bat`](https://github.com/rabbitmq/rabbitmq-server/blob/rabbitmq-server-691/scripts/rabbitmq-server.bat#L93)